src/Framework/src/Microsoft.AspNetCore.App.Runtime.csproj

```diff
@@ -0,0 +1,285 @@
+    <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
+    <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
+
+    <!-- Copy assets to output path from NuGet packages. -->
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+
+    <!-- Workaround for the way Microsoft.AspNetCore.Server.IIS.csproj references ANCM native deps. Do not copy content from ProjectReference's CopyToOutput items.  -->
+    <_GetChildProjectCopyToOutputDirectoryItems>false</_GetChildProjectCopyToOutputDirectoryItems>
+
+    <!-- This project should not be referenced via the `<Reference>` impementation. -->
+    <IsProjectReferenceProvider>false</IsProjectReferenceProvider>
+
+    <!-- Properties related to crossgen -->
+    <CrossGenSymbolsType>CreatePerfMap</CrossGenSymbolsType>
+    <CrossGenSymbolsType Condition="'$(TargetOsName)' == 'win'">CreatePDB</CrossGenSymbolsType>
+
+    <LibPrefix Condition=" '$(TargetOsName)' != 'win' ">lib</LibPrefix>
+    <LibExtension>.so</LibExtension>
+    <LibExtension Condition=" '$(TargetOsName)' == 'win' ">.dll</LibExtension>
+    <LibExtension Condition=" '$(TargetOsName)' == 'osx' ">.dylib</LibExtension>
+    <ExeExtension Condition=" '$(TargetOsName)' == 'win' ">.exe</ExeExtension>
+    <!-- 3B = semicolon in ASCII -->
+    <PathSeparator Condition="'$(PathSeparator)' == ''">:</PathSeparator>
+    <PathSeparator Condition=" '$(TargetOsName)' == 'win' ">%3B</PathSeparator>
+
+    <!-- Crossgen executable name -->
+    <CrossgenToolFileName>crossgen</CrossgenToolFileName>
+    <CrossgenToolFileName Condition=" '$(TargetOsName)' == 'win' ">$(CrossgenToolFileName).exe</CrossgenToolFileName>
+    <!-- Default crossgen executable relative path -->
+    <CrossgenToolPackagePath>$(CrossgenToolFileName)</CrossgenToolPackagePath>
+    <!-- Disambiguated RID-specific crossgen executable relative path -->
+    <CrossgenToolPackagePath Condition=" '$(TargetRuntimeIdentifier)' == 'linux-arm' ">x64_arm\$(CrossgenToolPackagePath)</CrossgenToolPackagePath>
+    <CrossgenToolPackagePath Condition=" '$(TargetRuntimeIdentifier)' == 'linux-arm64' ">x64_arm64\$(CrossgenToolPackagePath)</CrossgenToolPackagePath>
+    <CrossgenToolPackagePath Condition=" '$(TargetRuntimeIdentifier)' == 'win-arm' ">x86_arm\$(CrossgenToolPackagePath)</CrossgenToolPackagePath>
+
+    <RuntimePackageRoot>$(NuGetPackageRoot)/runtime.$(RuntimeIdentifier).microsoft.netcore.app/$(MicrosoftNETCoreAppPackageVersion)/</RuntimePackageRoot>
+    <CrossgenToolPath>$(RuntimePackageRoot)tools/$(CrossgenToolPackagePath)</CrossgenToolPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="@(Dependency);@(ExternalDependency)" />
+    <ProjectReference Condition="'$(BuildIisNativeProjects)' == 'true' AND '$(BuildNative)' != 'false'" Include="$(RepositoryRoot)src\Servers\IIS\AspNetCoreModuleV2\InProcessRequestHandler\InProcessRequestHandler.vcxproj">
```

Comment:
Why not reference IISReferenceResolution tasks from here and use NativeReference?

Reply:
IIS resolution ends up copying both x86 and x64 into output, and puts it under `$(outdir)/$(arch)/`. I want to reference one bitness and have it in `$(outDir)`.

Reply:
You are wrong about both platforms part, in master it references only one platform https://github.com/aspnet/AspNetCore/blob/master/eng/targets/ResolveIisReferences.targets#L13. Output location can be changed by setting LinkBase.

Reply:
You're right, looks like we have two `NativeProjectReference`, each with a platform. I still need slightly different properties though, like ReferenceOutputAssembly.